src/slices/newsSlice.tsx

```diff
@@ -0,0 +1,48 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from '../app/store';
+import FeedItem from '../interfaces/FeedItem';
+
+interface News {
+  count: number;
+  news: Array<FeedItem>;
```

Comment:
Лучше короткий синтаксис типизации массива
```suggestion
  news: FeedItem[];
```